docs/source/algorithms/LoadILLTOF-v1.rst

```diff
@@ -21,6 +21,23 @@ be loaded from the Vanadium data.
 
 To date this algorithm only supports: IN4, IN5 and IN6
 
+Pulse Intervals
```

Comment:
Seem to be. Would it be worth mentioning that the unit of the number 60 in the last equation is in seconds?